lib/tasks/data.rake

```diff
@@ -0,0 +1,12 @@
+def data_migrations
+  puts 'Running data migrations configured in lib/tasks/data.rake...'
+end
+
+namespace :data do
+  desc 'Migrates data'
+  task :migrate do
+    at_exit { data_migrations }
+  end
+end
+
+Rake::Task['db:migrate'].enhance(['data:migrate'])
```

Comment:
Although this works as expected this threw me off a bit since defining the task in `.enhance` will run it before the `db:migrate` but this has been mitigated by the `at_exit` in the task itself. Maybe we define the task in a block as follows to ensure it's more explicit it's intention is to run after? 
```
Rake::Task['db:migrate'].enhance do
  Rake::Task['data:migrate'].invoke
end
```